IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/IntelliTectTests.cs

```diff
@@ -25,8 +25,9 @@ public void Test1()
                     IsBodyAvailable = true
                 };
 
-            TestBuilder builder = new TestBuilder();
+            TestBuilder builder = new TestBuilder("Test1");
```

Comment:
There is a fancy option here that could be helpful. If you change the `TestBuilder` constructor to look like this:
```C#
public TestBuilder([CallerMemberName]string caller = null) 
{
    //Caller is automatically provide by the compiler
}
```

You can then make this line just: `TestBuilder builder = new TestBuilder();` and the compiler will automatically put in the name the method that you were in when you called this. Alternatively you can override this by specifying your own string.